R/create_target_design.R

```diff
@@ -177,3 +177,60 @@ target_types_conc <- c("peak", "cmax", "cmax_1hr", "trough", "cmin", "conc")
 is_on_target <- function(v, target) {
   v >= target$min & v <= target$max
 }
+
+
+#' Create evaluation object
+#'
+#' This function defines the evaluation metric and timing for non-target metrics.
+#' Use this function to record outputs from the simulation like troughs or AUC
+#' that are not at the target times.
+#'
+#' @inheritParams create_target_design
+#' @export
+create_eval_design <- function(
+    evaltype = mipd_target_types(),
+    time = NULL,
+    when = NULL,
+    offset = NULL,
+    at = NULL,
+    anchor = c("dose", "day")
+) {
+  evaltype <- match.arg(tolower(evaltype), mipd_target_types())
```

Comment:
do we want to allow multiple types of data to be reported within one simulation? might be a nice flexibility, with minimal code required to support it?